src/transformers/data/data_collator.py

```diff
@@ -344,6 +518,7 @@ class DataCollatorForSeq2Seq:
     max_length: Optional[int] = None
     pad_to_multiple_of: Optional[int] = None
     label_pad_token_id: int = -100
+    return_tensors: str = 'pt'
```

Comment:
Need to be passed below to the tokenizer.pad, no?

Reply:
Yep, this one was unfinished, sorry! It's done now.